frontend/src/components/ProjectTeacherComponent.tsx

```diff
@@ -51,137 +53,144 @@ export function ProjectTeacherComponent(props: { project: ProjectTeacher }): JSX
-                    <div style={{width: "33%"}}> {/* Deze moet er blijven, anders doet css raar*/}
-                        <textarea className="textarea" placeholder={t('create_project.description.placeholder')}
-                                  value={description}
-                                  onChange={e => setDescription(e.target.value)}/>
+                        }
                     </div>
                 </div>
-            </div>
-            {/* DOCKER FIELD */}
-            <div className="field is-horizontal">
-                <div className="field-label">
-                    <label className="label">{t('create_project.docker_file.tag')}</label>
+                {/* DESCRIPTION FIELD */}
+                <div className="field is-horizontal">
+                    <div className="field-label">
+                        <label className="label">{t('create_project.description.tag')}</label>
+                    </div>
+                    <div className="field-body field">
+                        <div style={{width: "33%"}}> {/* Deze moet er blijven, anders doet css raar*/}
+                            <textarea className="textarea" placeholder={t('create_project.description.placeholder')}
+                                    value={description}
+                                    onChange={e => setDescription(e.target.value)}/>
+                        </div>
+                    </div>
                 </div>
-                <div className="field-body field file has-name">
-                    <label className="file-label">
-                        <input className="file-input" type="file" name="resume"/>
-                        <span className="file-cta">
-                            <span className="file-icon">
-                                <FaUpload/>
+                {/* DOCKER FIELD */}
+                <div className="field is-horizontal">
+                    <div className="field-label">
+                        <label className="label">{t('create_project.docker_file.tag')}</label>
+                    </div>
+                    <div className="field-body field file has-name">
+                        <label className="file-label">
+                            <input className="file-input" type="file" name="resume"/>
+                            <span className="file-cta">
+                                <span className="file-icon">
+                                    <FaUpload/>
+                                </span>
+                                <span className="file-label">
+                                    {t('create_project.docker_file.choose_button')}
+                                </span>
                             </span>
-                            <span className="file-label">
-                                {t('create_project.docker_file.choose_button')}
+                            <span className="file-name">
+                                C:\home\files\docker_file.dockerfile

```

Comment:
Hier mss iets systeems agnostisch zetten..
bvb "no file selected" "geen bestand geselecteerd"
totdat er effectief wel is geselecteerd is.
werd deze pagina nog niet correct mss? Want als ik iets selecteer werkt het gewoon niet?

Reply:
Dit is niet mijn code maar komt van de merge met main. Ik heb enkel de create-project-topbar van ProjectViewTeacher naar deze file veranderd om de scrollbar hier ook over te laten lopen. Hierdoor moest ik wel een div rond de code plaatsen en is de indentatie van heel de file aangepast. Git markeerd dit als veranderd en vandaar ook dat heel de file aangepast lijkt maar eigenlijk toch niet.